<launch>
<!--launch di tutti e 3 i pkg-->
<include file="$(find pc_wp)/launch/launch.launch"/>

<!--Nodo che valuta l'errore e l'MSE-->
<node name="performance_analysis" pkg ="nav_pkg" type="performance_analysis"/> 

<!--Plot degli errori sulle comp. delle variabili di stato-->
<node name="rpy_plot" pkg="rqt_plot" type="rqt_plot" args="/error/rpy"/>
<node name="pos_plot" pkg="rqt_plot" type="rqt_plot" args="/error/pos"/>
<node name="vel_plot" pkg="rqt_plot" type="rqt_plot" args="/error/lin_vel"/>
<node name="MSE_plot" pkg="rqt_plot" type="rqt_plot" args="/error/MSE"/>
</launch>